examples/bert/README.md

```diff
@@ -14,6 +14,12 @@ The workflow in [bert_ptq_cpu_aml.json](bert_ptq_cpu_aml.json) is similar to the
 In order to use this example, `<place_holder>` in the `azureml_client` section must be replaced with the appropriate values for your
 AzureML workspace.
 
+#### Huggingface Model Optimization
+The workflow in [hf_bert_ptq_cpu.json](hf_bert_ptq_cpu.json) is similar to the above workflow, but uses

```

Comment:
Suggest changing the title to BERT optimization with Huggingface configurations.

Reply:
Updated.